docs/CHANGELOG.rst

```diff
@@ -47,6 +49,8 @@ Fixed
 - Improve handling of validation errors
 - Ignore file size in assertFields
 - Order data objects in ``CollectionViewSet``
+- Fix tests for Django 1.10
+- Add quotes to a test paths
```

Comment:
Make it a bit more descriptive, maybe: Add quotes to file paths in test processes